rotkehlchen/tests/api/test_history_events_export.py

```diff
@@ -121,6 +122,31 @@ def test_history_export_download_csv(
     assert_csv_export_response(response, download_dir, is_download=True)
 
 
+@pytest.mark.vcr
+@pytest.mark.parametrize('should_mock_price_queries', [False])
+@pytest.mark.parametrize('db_settings', [
+    {'csv_export_delimiter': ';'},
+])
+def test_history_export_csv_custom_delimiter(
+        rotkehlchen_api_server_with_exchanges,
+        tmpdir_factory,
+):
+    """Test that using a custom csv delimiter works correctly."""
+    database = rotkehlchen_api_server_with_exchanges.rest_api.rotkehlchen.data.db
+    add_entries(events_db=DBHistoryEvents(database=database))
+
+    with database.conn.read_ctx() as cursor:
+        csv_delimiter = database.get_settings(cursor).csv_export_delimiter
+
+    csv_dir = str(tmpdir_factory.mktemp('test_csv_dir'))
+
```

Comment:
nitpick. no reason for newline here

```suggestion
    csv_dir = str(tmpdir_factory.mktemp('test_csv_dir'))
```